share/dynamo-to-share/template.yml

```diff
@@ -0,0 +1,63 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: 'AWS::Serverless-2016-10-31'
+Description: Publish data from a Amazon DynamoDB Stream to a Vendia Share node
+
+Parameters:
+
+  ShareNodeUrl:
+    Type: String
+  
+  ShareNodeApiKey:
+    Type: String
+    NoEcho: true
+
+Globals:
+  Function:
+    Runtime: python3.8
+    Timeout: 60
+    Handler: index.handler
+    MemorySize: 256
+
+Resources:
+  # Amazon DynamoDB Table to store inventory data
+  InventoryTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      AttributeDefinitions:
+        - AttributeName: item_number
+          AttributeType: "S"
+      KeySchema:
+        - AttributeName: item_number
+          KeyType: HASH
+      StreamSpecification:
+        StreamViewType: NEW_IMAGE
+      ProvisionedThroughput:
```

Comment:
Instead of provisioned, please use PAY_PER_REQUEST so that we don't consume any capacity that isn't used.